### ISSUE TITLE
Load from JSON not work!

### ISSUE THREAD

**Issue by CaMMMXX** [fkunn1326/openpose-editor#121]
**Describe the bug**
Uploading a JSON file which is generated from ControlNet, then nothing happens

**To Reproduce**
Press the Load from JSON button, select a JSON file, click OK, and nothing happens

**Expected behavior**
It should run as normal and display the pose line

**Environment**
 - macOS 13.4 M1 Max
 - chrome

**Additional context**
version: v1.2.1  •  python: 3.10.11  •  torch: 2.1.0.dev20230523  •  xformers: N/A  •  gradio: 3.29.0  •  checkpoint: 5493a0ec49

**Comment by naduarte2** [comment]
Same here, I generated a pose, send it to txt2img ControlNet 0, but it wasn't recognized, even loading the JSON pose I created, it shows the error:

```python
TypeError: Cannot read properties of undefined (reading 'map')
```

**Comment by LaBelgique** [comment]
Same problem here

> **Describe the bug** Uploading a JSON file which is generated from ControlNet, then nothing happens
> 
> **To Reproduce** Press the Load from JSON button, select a JSON file, click OK, and nothing happens
> 
> **Expected behavior** It should run as normal and display the pose line
> 
> **Environment**
> 
>     * macOS 13.4 M1 Max
> 
>     * chrome
> 
> 
> **Additional context** version: v1.2.1  •  python: 3.10.11  •  torch: 2.1.0.dev20230523  •  xformers: N/A  •  gradio: 3.29.0  •  checkpoint: 5493a0ec49

Same problem here, any fix ?

**Comment by icecreamzhao** [comment]
same question too on windows 11

**Comment by palentir** [comment]
it's why I love this community. I had this issue last week and I suddently remembered it now, so I came here to find out if anyone else had same issue. Then some guy finds the 1 line causing it and the dev updated it 4 days ago. That is awesome.. So I went to check if mine was already fixed and it was :-) #1 pose editor <3
